setup.py

```diff
@@ -54,21 +55,38 @@
 test_reqs = ["pytest", "pytest-cov"]
 
 
-def get_version():
-    VERSIONFILE = join("pymc", "__init__.py")
-    lines = open(VERSIONFILE).readlines()
+def get_distname(nightly_build=False):
+    distname = "pymc"
+    if nightly_build:
+        distname = f"{distname}-nightly"
+
+    return distname
+
+
+def get_version(nightly_build=False):
+    version_file = join("pymc", "__init__.py")
+    lines = open(version_file).readlines()
     version_regex = r"^__version__ = ['\"]([^'\"]*)['\"]"
     for line in lines:
         mo = re.search(version_regex, line, re.M)
         if mo:
-            return mo.group(1)
-    raise RuntimeError(f"Unable to find version in {VERSIONFILE}.")
+            version = mo.group(1)
+
+            if nightly_build:
+                suffix = datetime.now(timezone.utc).strftime(r".dev%Y%m%d")
+                version = f"{version}{suffix}"
+
+            return version
+
+    raise RuntimeError(f"Unable to find version in {version_file}.")
```

Comment:
nit: Move this up top with the other globals on line 23 or so and capitalize it to make it clear it is a global

Reply:
I'm confused. What would you like me to move up.?`version` and `version_file` are now within the function so they aren't global

Reply:
I'm sorry I meant this line with the `nightly` global var

https://github.com/pymc-devs/pymc/pull/5490/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R84